Cargo.toml

```diff
@@ -236,7 +236,7 @@ aws-sdk-secretsmanager = { version = "1.6.0", default-features = false, features
 aws-sdk-sts = { version = "1.3.1", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
 aws-types = { version = "1.3.5", default-features = false, optional = true }
 aws-sigv4 = { version = "1.2.9", default-features = false, features = ["sign-http"], optional = true }
-aws-config = { version = "~1.0.1", default-features = false, features = ["behavior-version-latest", "credentials-process", "sso", "rt-tokio"], optional = true }
+aws-config = { version = "1.1.9", default-features = false, features = ["behavior-version-latest", "credentials-process", "sso", "rt-tokio"], optional = true }
```

Comment:
Minimum version necessary to be able to read the endpoint_url field.

Reply:
And it looks like v1.1.8 is the minimum version to support #20662. Thanks for doing this!